docs/src/main/sphinx/connector/faker.md

```diff
@@ -0,0 +1,130 @@
+# Faker connector
+
+The Faker connector generates realistically looking random data matching a
+specific schema.
+
+## Configuration
+
+To configure the Faker connector, create a catalog properties file
```

Comment:
The connector can create data in Trino and allowing querying it. Create a catalog
properties file that specifies the Faker connector by setting the
`connector.name` to `faker`.

For example, to access data as the `example` catalog, create the file
`etc/catalog/example.properties`. Replace the connection properties as
appropriate for your setup: